### ISSUE TITLE
[remark-typo]: french quotes not inserted on some WTF cases

### ISSUE THREAD

**Issue by artragis** [zestedesavoir/zmarkdown#469]
Here is the sentence : 
```
Les propriétaires de l'application créent un <<contenu>>, lui donne des <<méta données>> et les visiteurs vont pouvoir accéder à la fiche ainsi rédigée de ces contenus.
```
Here is the result : 

```
Un CMS distingue deux types d’utilisateurs : les propriétaires et les visiteurs. Le concept est le suivant : Les propriétaires de l’application créent un <<contenu>>, lui donne des «méta données» et les visiteurs vont pouvoir accéder à la fiche ainsi rédigée de ces contenus.
```

the "contenu" part is not properly translated.
It is translated if we put a blanck space between `<<`  and  `c`.

It is expected not to have to put this extra space.

**Comment by artragis** [comment]
Another one is when the << >> elements are inside a list element.

**Comment by AmauryCarrade** [comment]
Also for the `<<méta données>>` part (i.e. when characters are correctly replaced), inner spaces are not inserted; I think they should be, so to respect French typography.

**Comment by StaloneLab** [comment]
I recently investigated this problem, which was more complicated than I expected, and am left with the following:

- allowing the syntax without space would be complicated, because of how remark works internally, and of how the plugin was designed (visiting the tree after tokenization). I am not ready to rewrite this plugin (or any other) without a micromark migration (because the job would be done twice), but anyway I take note of this issue;
- what I suggest is therefore to enforce the following syntax: `<< [content] >>`, with spaces inside, that will be replaced by NBSP. It would solve the issue in the following way: `<<méta données>>` will give the same result as `<<contenu>>`: no french quotes.

**Comment by artragis** [comment]
good to me. The issue was coming from inconsistancy, so it would solve it.

**Comment by StaloneLab** [comment]
Okay, finally we should be able to solve it correctly. I found one of our plugins, `remark-fix-guillemets`, which already does what we want. It was certainly forgotten during one of the refactorings. I included it in my PR for ZMarkdown 11.2. The problem reported by @AmauryCarrade about adding NBSP was also solved by a simple fix to the plugin.
